src/components/BaseCard.vue

```diff
@@ -46,7 +46,10 @@
           </div>
         </div>
         <router-link
-          v-if="+get(post, 'relationships.creator.id', null) === +userState.id"
+          v-if="
+            +get(post, 'relationships.creator.id', null) === +userState.id ||
```

Comment:
Longer expressions in templates should be extracted to computed properties for easier debugging